docs/source/guides/index.mdx

```diff
@@ -122,13 +122,14 @@ If you register a persisted query with a specific client name instead of `null`,
 
 Use the `--header` option when running the MCP Server to pass the header to the router. The default name of the header expected by the router is `apollographql-client-name`. To use a different header name, configure `telemetry.apollo.client_name_header` in router YAML configuration.
 
-### From schema introspection
+## Introspection tools
```

Comment:
Please make sure to update the following links if you want to change this heading.

https://github.com/apollographql/apollo-mcp-server/blob/2ace069380914a57d849b66c57c4c96ad1ae9f29/docs/source/best-practices.mdx?plain=1#L21
https://github.com/apollographql/apollo-mcp-server/blob/2ace069380914a57d849b66c57c4c96ad1ae9f29/docs/source/command-reference.mdx?plain=1#L107
https://github.com/apollographql/apollo-mcp-server/blob/2ace069380914a57d849b66c57c4c96ad1ae9f29/docs/source/quickstart.mdx?plain=1#L186